changelogs/current.yaml

```diff
@@ -184,6 +184,10 @@ new_features:
 - area: tcp_proxy
   change: |
     added an option to dynamically disable TCP tunneling even if set in the filter config, by setting a filter state object for the key ``envoy.tcp_proxy.disable_tunneling``.
+- area: tcp_proxy
+  change: |
+    add :ref:`flush access log on connected <envoy_v3_api_field_extensions.filters.network.tcp_proxy.v3.TcpProxy.flush_access_log_on_connected>` to allow flushing access log state
```

Comment:
Flush is confusing in the general context since it might mean IO flush.
How about: ```add an option to record an access log entry on the connection open event```?

Can you also clarify the relationship with the item below for period logging? Does this require period logging enabled?

Reply:
done